editoast/src/generated_data/sql/generate_psl_sign_layer.sql

```diff
@@ -30,36 +33,79 @@ WITH signs AS (
 collect AS (
     SELECT signs.sc_id,
         signs.data,
+        tracks_layer.geographic AS track_geo,
+        tracks_layer.schematic AS track_sch,
+        LEAST(
+            GREATEST(position / (tracks.data->>'length')::float, 0.),
+            1.
+        ) AS norm_pos,
         ST_LineInterpolatePoint(
             tracks_layer.geographic,
             LEAST(
-                GREATEST(
-                    signs.position / (tracks.data->'length')::float,
-                    0.
-                ),
+                GREATEST(position / (tracks.data->>'length')::float, 0.),
                 1.
             )
-        ) AS geo,
+        ) as geo,
```

Comment:
minor: Any reason why making them lowercase. All of the other SLQ keywords seems to be uppercase, I propose to keep it that way?

Reply:
No reason you are right 😄